src/tabs/tab-header-bar.tsx

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
is the plan to add `role="application"` to this PR?

Reply:
Yes, Sterly wanted a general review first, just to gauge what the extent of the feedback from us will be 😅 So next step would be addressing the comments and implementing the role=application approach.